Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -363,10 +363,14 @@ def _wait_until_device_in_state(device, state, deadline):
     @staticmethod
     def _wait_until_device_is_usable(device, deadline):
         _log.debug(u'Waiting until {} is usable'.format(device))
-        while not device.platform_device.is_usable(force_update=True):
+        state = device.platform_device.is_usable(force_update=True)
+        while not state:
+            if state is None:
+                raise RuntimeError(u'{} will never become healthy'.format(device))
```

Comment:
nit: We should never be running this on any python version below 3.9, so you can modernize the string if you want.
```suggestion
                raise RuntimeError(f'{device} will never become healthy')
```